pkg/handler/processor/guesser/type_guesser.go

```diff
@@ -25,6 +25,7 @@ func init() {
 	_ = RegisterDocumentTypeGuesser(&ite6TypeGuesser{}, "ite6")
 	_ = RegisterDocumentTypeGuesser(&dsseTypeGuesser{}, "dsse")
 	_ = RegisterDocumentTypeGuesser(&spdxTypeGuesser{}, "spdx")
+	_ = RegisterDocumentTypeGuesser(&cycloneDXTypeGuesser{}, "cyclonedx")
```

Comment:
can we add a test case to `type_guesser_test.go` as well to include a CDX document?

Reply:
done!